source/MaterialXGenShader/GenOptions.h

```diff
@@ -149,6 +150,11 @@ class MX_GENSHADER_API GenOptions
     /// Enables the writing of a directional albedo table.
     /// Defaults to false.
     bool hwWriteAlbedoTable;
+
+    /// Enable emitting colorspace transform code if a color management.
+    /// system is defined.
+    /// Default is set to true.
+    bool emitColorTransforms;
```

Comment:
New option. Allows for existing setup where a color management system may be specified but allows for either to emit shader code or not. Alternative is to set / reset colormanagement system but this is more cumbersome. Also it's hard to do for web which has not exposed cm systems yet.